lib/facter/conjur.rb

```diff
@@ -61,7 +61,7 @@ def version
 
     def authenticate url, certs, credentials
       login, key = credentials
-      uri = URI(url + '/') + authentication_path(login)
+      uri = URI(url) + authentication_path(login)
```

Comment:
This concerns me a little bit, because leaving off a trailing slash is a configuration mistake that would easy to make.

How would the user know that this is what they did wrong?

Are there any alternative approaches that could be used here? (E.g. `url += '/' unless url.end_with? '/'`, `URI::join`, etc.)

Reply:
`URI.join` won't work -- with a file path it'll snip off the last segment. But perhaps you're right I should accommodate this.